src/components/editMeasure/testCases/components/editTestCase/calculator/durationTab/DurationTab.test.tsx

```diff
@@ -71,4 +71,113 @@ describe("DurationTab", () => {
     fireEvent.click(checkbox);
     expect(checkbox).toBeChecked();
   });
+
+  it("shows '--' for results if calculate is clicked without both dates", () => {
+    render(<DurationTab />);
+    const calculateButton = screen.getByTestId("calculate-duration");
+    fireEvent.click(calculateButton);
```

Comment:
`userEvent` is better than fireEvent it seems because it simulates interaction  by triggering sequence of events that would be triggered if real user interacts with element

Reply:
done I have made the changes, thanks for the feedback